package.json

```diff
@@ -5,17 +5,17 @@
   "main": "index.js",
   "dependencies": {
     "execspawn": "^1.0.1",
-    "minimist": "^1.2.0",
-    "mkdirp": "^0.5.1",
-    "node-abi": "^2.0.0",
-    "npm-run-path": "^3.1.0",
+    "mkdirp": "^1.0.4",
+    "node-abi": "^2.19.1",
+    "npm-run-path": "^4.0.1",
+    "minimist": "^1.2.5",
     "pump": "^3.0.0",
     "tar-fs": "^2.1.0"
   },
   "devDependencies": {
-    "nan": "^2.5.1",
     "standard": "^13.0.1",
-    "tape": "^4.6.3"
+    "tape": "^4.6.3",
+    "nan": "^2.14.2"
```

Comment:
why did the ordering change here?

Reply:
npm automatically sorts the deps based on the alphabet. But this one seems like a bug in their sorting!

Reply:
Fixed in https://github.com/prebuild/prebuildify/pull/45/commits/2bfeaef2e389f1018e844e2272d6976361fda4b4